components/src/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,282 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { Manager, Reference, Popper } from "react-popper";
+import theme from "../theme";
+/* eslint react/destructuring-assignment: 0 */
+
+const tooltipStyles = {
+  backgroundColor: theme.colors.white,
+  borderColor: theme.colors.grey,
+  textColor: theme.colors.black,
+};
+
+const getTooltipMargin = placement => {
+  const direction = String(placement).split("-")[0];
+  switch (direction) {
+    case "bottom":
+      return ({
+        marginTop: "4px",
+      });
+    case "top":
+      return ({
+        marginBottom: "4px",
+      });
+    case "right":
+      return ({
+        marginLeft: "4px",
+      });
+    case "left":
+      return ({
+        marginRight: "4px",
+      });
+    default:
+      return ({});
+  }
+};
+
+const TooltipContainer = styled.div({
+  color: tooltipStyles.textColor,
+  display: "flex",
+  flexDirection: "column",
+  fontSize: "14px",
+  backgroundColor: tooltipStyles.backgroundColor,
+  borderRadius: theme.radii.medium,
+  border: `1px solid ${tooltipStyles.borderColor}`,
+  boxShadow: "0 2px 4px rgba(0, 0, 0, 0.18)",
```

Comment:
Is this going to be a common pattern?
I see the same boxShadow being applied to `MenuItem`.